docs/guides/install-python.md

```diff
@@ -54,12 +60,12 @@ Even if a specific Python version is not requested, uv will download the latest
 $ uv venv --python-preference only-managed
 ```
 
+<!-- TODO(zanieb): Restore when Python shim management is added
 Note that when an automatic Python installation occurs, the `python` command will not be added to the shell. Use `uv python install-shim` to ensure the `python` shim is installed.
+-->
 
 ## Using an existing Python installation
 
-uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior, uv will use the system Python if it satisfies the requirements of the command invocation.
-
-To force uv to use the system Python, provide the `--python-preference only-system` option.
+uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior, uv will use the system Python if it satisfies the requirements of the command invocation. See the [Python discovery](../python-versions.md#discovery-order) documentation for details.
```

Comment:
I think the comma after behavior is a little off. Maybe a semicolon or em dash?

Reply:
```suggestion
uv will also use an existing Python installation if already present on the system. There's no configuration necessary for this behavior: uv will use the system Python if it satisfies the requirements of the command invocation. See the [Python discovery](../python-versions.md#discovery-order) documentation for details.
```